dns/UnboundBL/src/opnsense/service/templates/OPNsense/UnboundBL/rc.conf

```diff
@@ -0,0 +1,2 @@
+whitelist="/({{ OPNsense.UnboundBL.general.whitelist.replace(',', '|') }})/"
```

Comment:
You have to set a check if != '' and then the content, otherwise you get an error when it's empty. 
Just look in the bind plugin the templating stuff .. most cases should be there. Also the telegraf plugin offers most templating which you need.